homeassistant/components/iotmeter/config_flow.py

```diff
@@ -0,0 +1,63 @@
+"""Config flow for IoTMeter integration."""
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+
+from .const import DOMAIN
+
+
+class EVConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for IoTMeter."""
+
+    async def async_step_user(self, user_input=None):
+        """Handle the initial step."""
+        errors = {}
+        if user_input is not None:
+            ip_address = user_input.get("ip_address")
+            port = user_input.get("port")
+
+            if ip_address and port:
+                return self.async_create_entry(title="IoTMeter", data=user_input)
```

Comment:
We should check if the data is valid. The config flow is the best way to tell the user they did something wrong, so we should try to connect with the device to verify the data